edit-post/assets/stylesheets/_z-index.scss

```diff
@@ -55,6 +55,8 @@ $z-layers: (
 	'.components-popover.is-bottom': 99990,
 
 	'.components-autocomplete__results': 1000000,
+
+	'.skip-to-selected-block': 100000,
```

Comment:
Why so high? In fact, even without `z-index` at all, it appears to layer as I'd expect.

FWIW, this isn't the same as putting the button on equal footing as `.edit-post-sidebar`, since it's already within the stacking context of the sidebar.

Reply:
For re-use in other places e.g. header toolbar, publishing flow, etc. I guess it should have a high z-index.